docs/src/main/sphinx/release/release-467.md

```diff
@@ -0,0 +1,40 @@
+# Release 467 (5 Dec 2024)
+
+## General
+
+* Change default protocol for internal communication to HTTP/1.1 to address
+  issues with HTTP/2. ({issue}`24281`)
+* Return compressed results to clients by default when using the spooling
+  protocol. ({issue}`24332`)
+* Add application identifier `gcs.application-id`, `azure.application-id` or
+  `s3.application-id` to the storage when using the spooling protocol. ({issue}`24361`)
+* Add support for OpenTelemetry tracing to the HTTP, Kafka, and MySQL event
+  listener. ({issue}`24389`)
+* Fix query failures or missing statistics in `SHOW STATS` when a connector
+  returns `NaN` values for table statistics. ({issue}`24315`)
+
+## Docker image
+
+* Remove the `microdnf` package manager.  ({issue}`24281`)
+* Fix graceful shutdown. ({issue}`24380`)
```

Comment:
Not sure if this should be here or in the general section @wendigo @nineinchnick @martint